src/Api/KeyManagement/Models/Response/PrivateAccountKeysResponseModel.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Bit.Core.KeyManagement.Models.Data;
+using Bit.Core.Models.Api;
+
+namespace Bit.Api.Models.Response;
+
+/// <summary>
+/// This response model is used to return keys of a user - downstream of the user key - to the client.
+/// This includes the private keys (signature/encryption), and proof tying one to another. This could
+/// also be used to contain further user-owned keys in the future (per-vault keys, etc). This should
+/// not be used to contain keys not just owned by the user (e.g. organization keys).
+/// </summary>
+public class PrivateAccountKeysResponseModel : ResponseModel
+{
+    public PrivateAccountKeysResponseModel(UserAccountKeysData signingKeys) : base("accountKeys")
+    {
+        if (signingKeys != null)
+        {
+            SigningKeys = signingKeys.SigningKeyData;
+        }
+        AsymmetricEncryptionKeys = signingKeys.AsymmetricEncryptionKeyData;
```

Comment:
This can throw a null reference exception when `signingKeys` is null. 

It's unclear to me what the expectation here is, would signingKeys ever be null and if so what is the desired result.

Reply:
Oh, this is just wrong, whoops. Will be corrected in the split-off follow-up pr.